### PR TITLE
LilyPad PS1 analog mode fix and UI improvement

### PR DESCRIPTION
**Pull request by FlatOutPS2**
Fixes issue where "Use analog mode if possible - PS1 only" would enable
analog mode when playing games that don't support it, as described in this issue: https://github.com/PCSX2/pcsx2/issues/1348

And improves UI:

**Master Branch**:
lilypad ui current

**Pull Request Branch**:
lilypad pullrequest2

The layout of the buttons is improved to more closely resemble a modern analog controller/DualShock configuration, and the Device column of the list view has been reduced slighty so by default the horizontal scroll bar isn't visible.

### CODE REVIEW

File: plugins/LilyPad/LilyPad.rc [hunk 1]
```diff
@@ -58,82 +58,84 @@ BEGIN
...
+	LTEXT           "Dead Zone", IDC_LABEL_DEADZONE, 202, 243, 42, 8
+	CONTROL         "", IDC_SLIDER_DEADZONE, "msctls_trackbar32", WS_TABSTOP, 240, 239, 131, 17
+	EDITTEXT        IDC_AXIS_DEADZONE, 377, 242, 33, 12, ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP, WS_EX_RTLREADING
+	PUSHBUTTON      "Lock Input", ID_LOCK, 196, 263, 59, 15
+	PUSHBUTTON      "Lock Direction", ID_LOCK_DIRECTION, 196, 281, 59, 15
+	PUSHBUTTON      "Lock Buttons", ID_LOCK_BUTTONS, 196, 299, 59, 15
+	GROUPBOX        "Add Force Feedback Effect", IDC_STATIC, 262, 263, 155, 51
+	COMBOBOX        IDC_FORCEFEEDBACK, 269, 276, 142, 106, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+	PUSHBUTTON      "Big Motor", ID_BIG_MOTOR, 269, 294, 64, 14
+	PUSHBUTTON      "Small Motor", ID_SMALL_MOTOR, 347, 294, 64, 14
```
Comment (ssakash) [inline plugins/LilyPad/LilyPad.rc]: Hand edited ? 

According to **Rule #7834** on turtleli's book , spaces between other arguements should be removed on resource script files.

**Example**:
``   "Small Motor", ID_SMALL_MOTOR, 347, 294, 64, 14``  **->**   ``Small Motor",ID_SMALL_MOTOR,347,294,64,14``

**Comment by FlatOutPS2** [comment]
I copied them to from a test build, VS silently added the spaces automatically when I pasted them. There weren't any spaces in the test build. >_<

File: plugins/LilyPad/LilyPad.rc (hunk 1 above)
Comment (FlatOutPS2) [inline plugins/LilyPad/LilyPad.rc]: I copied them to from a test build, VS silently added the spaces automatically when I pasted them. There weren't any spaces in the test build. >_<

**Comment by turtleli** [comment]
It's probably better to split your commit in 2 so you have an analog mode fix commit and a UI change commit.

Side note: Somehow both you and ssakash like to make the buildbots really busy...
untitled

**Comment by FlatOutPS2** [comment]
> Side note: Somehow both you and ssakash like to make the buildbots really busy...

I noticed a small mistake right after I uploaded the PR, but forgot to merge before pushing it. My bad. :p

**Comment by turtleli** [comment]
Code seems fine to me, I'll merge in a few days unless all hell breaks loose on this PR cause, yeah, GUI.

File: plugins/LilyPad/LilyPad.rc [hunk 2]
```diff
@@ -58,82 +58,84 @@ BEGIN
...
+    CONTROL         "Flip",IDC_FF_AXIS5_FLIP,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,302,207,35,10
+    EDITTEXT        IDC_FF_AXIS5_SCALE,375,207,33,12,ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP,WS_EX_RTLREADING
+    CONTROL         "",IDC_FF_AXIS6,"msctls_trackbar32",WS_TABSTOP,199,223,214,17
+    CONTROL         "Axis 6",IDC_FF_AXIS6_ENABLED,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,205,243,91,10
+    CONTROL         "Flip",IDC_FF_AXIS6_FLIP,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,302,243,35,10
+    EDITTEXT        IDC_FF_AXIS6_SCALE,375,243,33,12,ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP,WS_EX_RTLREADING
+    PUSHBUTTON      "Test",ID_TEST,196,263,59,15
+    PUSHBUTTON      "Back to Controls",ID_CONTROLS,196,299,59,15
+    GROUPBOX        "Configure Binding",ID_SENSITIVITY,195,189,222,70
+    EDITTEXT        IDC_AXIS_DEVICE1,202,202,74,12,ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP //+7
```
Comment (ssakash) [inline plugins/LilyPad/LilyPad.rc]: Is this comment needed for some reason ?

Comment (FlatOutPS2) [inline plugins/LilyPad/LilyPad.rc]: Nope, not at all. :p

**Pull request merged**